### PR TITLE
Document DNS delegation requests

### PR DESCRIPTION
**Pull request by ChrisBAshton**
Signpost the SRE docs while we're expected to process these requests.

### CODE REVIEW

File: source/manual/zendesk.html.md [hunk 1]
```diff
@@ -94,6 +94,22 @@ some of the common data.gov.uk (DGU) tickets that come in and how to fix them.
...
+
+The requests will look something like the following:
+
+> Please delegate the following to `something.service.gov.uk`
+> `nameserver1.example.com`
+> `nameserver2.example.com`
+
+We should process these as per the [SRE interruptible documentation](https://docs.google.com/document/d/1QzxwlN9-HoewVlyrOhFRZYc1S0zX-pd97igY8__ZLAo/edit#heading=h.wg0s4ugkpdpc).
+
+The workflow for these requests is that a requester emails a particular email address, which creates a Zendesk ticket for the Policy & Engagement team. The request is then signed off and routed to us. We then double-check with someone from Policy & Engagement that the change has been agreed, then add/change the necessary DNS records (see [example](https://github.com/alphagov/govuk-dns-config/pull/854/files)) and respond to the requestor via the ticket.
```
Comment (nicholsj) [inline source/manual/zendesk.html.md:109]: Should be "GOV.UK Policy and Strategy team"

File: source/manual/zendesk.html.md [hunk 2]
```diff
@@ -94,6 +94,22 @@ some of the common data.gov.uk (DGU) tickets that come in and how to fix them.
...
+
+Note that some requests come directly via the [hostmaster Google group](https://groups.google.com/a/digital.cabinet-office.gov.uk/g/hostmaster). If you are in any doubt about the legitimacy of a request, reassign the ticket to `3rd Line--Policy and Engagement` and add an internal note asking them.
```
Comment (nicholsj) [inline source/manual/zendesk.html.md:111]: Zendesk group is `3rd Line -- GOV.UK Policy and Strategy`

**Review by nicholsj — COMMENTED** [review]
Note there's also this https://docs.publishing.service.gov.uk/manual/dns.html#requests-for-domain-names-and-other-dns-changes

File: source/manual/zendesk.html.md [hunk 3]
```diff
@@ -94,6 +94,22 @@ some of the common data.gov.uk (DGU) tickets that come in and how to fix them.
 For more thorny issues, respond to the user that you are passing this on for further
 review and assign to the `3rd Line--GOV.UK Product Requests` Zendesk group.
 
+## DNS delegation tickets
+
+As of October 2022, 2nd line developers may be asked to process DNS delegation requests. This used to be handled by a dedicated `3rd Line--GDS Reliability Engineering` Zendesk queue, which has now been merged into `2nd Line--GOV.UK Alerts and Issues`.
```
Comment (nicholsj) [inline source/manual/zendesk.html.md:99]:
```suggestion
As of October 2022, Technical 2nd line may be asked to process DNS delegation requests. This used to be handled by a dedicated `3rd Line--GDS Reliability Engineering` Zendesk queue, which has now been merged into `2nd Line--GOV.UK Alerts and Issues`.
```

Comment (nicholsj) [reply source/manual/zendesk.html.md:99]: Just for consistency

**Review by hannako — COMMENTED** [review]
This looks great! I just wondered if we should consolidate the location of the instructions? 

We link out to the SRE docs, but then we give some of the steps below but not all of them - we don't go into the need to plan and apply in aws and gcp via the jenkins jobs, but we do give a link to a pr. 

Is it better to have all of the steps in one place? 

Also, on the dns_deploy jenkins job there is a note:

```
This is required for the GCP provider. Generate a token with using the gcloud CLI (https://cloud.google.com/sdk/docs/quickstart#mac). Run: gcloud config set project govuk-production; gcloud auth login --brief; gcloud auth print-access-token
```

That didn't work for me until I'd installed the glcoud cli and I had to run a slightly different command. Should that be added in somewhere too? 



Either put the sample PR in the SRE docs, or copy the SRE docs into here?

**Review by AgaDufrat — APPROVED** [review]
Thanks for signposting to the Google doc with detailed instructions (which as you mentioned should not all be public). I think this is enough info for this page. I'll add more hints and details to the document.

**Comment by ChrisBAshton** [comment]
Thanks for the review, @nicholsj - text tweaked 👍 

Good points, @hannako. More context available in the [Slack thread](https://gds.slack.com/archives/C013F737737/p1666631618301629) - some of the content of that document is a little sensitive to move into the Dev Docs, so the aim was really to signpost to the SRE docs. But also to provide enough keywords that a search of the Dev Docs would bring this signposting up more easily.

Anything that's missing from the SRE docs should be added (thanks Aga for volunteering). Richard is also going to be reviewing/updating the SRE docs as a [post-incident action](https://docs.google.com/document/d/1tALS9dOii_frU1qCgHQo1xpPI_8xG1mL1l_48BU-eCs/edit?disco=AAAAiVDACs8), so hopefully the content will improve.

**Pull request merged**